hearth/lib/hearth/retry/retry_backoff_strategy.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Hearth
+  module Retry
+    # @api private
+    # Computes the next backoff delay for a retry attempt.
+    class RetryBackoffStrategy
```

Comment:
I don't like this class. I'd rather have a "call" interface and make it a proc (or a class) like we have backoff currently.